docs/link-flow.md

```diff
@@ -19,3 +19,18 @@ limitations under the License.
 This flow is normally originated from another surface and allows the reader to link this publication's subscription to that surface. See [Subscriptions APIs](./core-apis.md).
 
 The link flow would normally be executed by a 3p surface to let a user to claim an existing subscription. However, SwG client provides `linkAccount` and `setOnLinkComplete` APIs for testing purposes.
+
+# SwG Link Save flow (Not launched)
+
+This flow is normally originated from the publisher and allows the reader to link publication's subscription to the reader's account.
+
+To start saving subscription link, provide a request containing a token corresponding to the reader's
+subscription, Eg: {token: 'THE TOKEN'}
```

Comment:
Not quite. I meant actually use `{token: 'THE TOKEN'}` in the example below inside the code section. E.g.

```
subscriptions.saveSubscription({token: 'THE TOKEN'});
```